pkg/skaffold/util/util.go

```diff
@@ -341,3 +342,28 @@ func IsSubPath(basepath string, targetpath string) bool {
 func hasHiddenPrefix(s string) bool {
 	return strings.HasPrefix(s, hiddenPrefix)
 }
+
+// ShowHumanizeTime returns time in human readable format
+func ShowHumanizeTime(start time.Time) string {
```

Comment:
Can we make this take ```time.Duration``` instead of ```time.Time``` so that the test don't have any variance?

Reply:
I'll try that.